examples/hello-world/index.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```

Comment:
This is the simplest Hello World program I could think of, but if there's something else we'd like to express with the Hello World program, I'm happy to change this.

Reply:
Embedding a dependency would be nice. For example, `chalk`